apps/trustlab/src/payload/collections/Posts.js

```diff
@@ -40,7 +46,39 @@ const Posts = {
         hidden: false,
       },
     }),
+    image({
+      overrides: {
+        name: "image",
+        required: true,
+      },
+    }),
+    {
+      name: "excerpt",
+      type: "textarea",
+      localized: true,
+      required: true,
+      admin: {
+        position: "sidebar",
+      },
+    },
+    richText({
+      name: "content",
+      localized: true,
+    }),
+    {
+      name: "tags",
+      type: "relationship",
+      relationTo: "tags",
+      hasMany: true,
+      localized: true,
+      admin: {
+        position: "sidebar",
+      },
+    },
   ],
+  hooks: {
+    afterRead: [nestCollectionUnderPage("posts")],
+  },
```

Comment:
I don't think all posts will be under posts as some will need to be under events, opportunities, etc.